src/lwc-dev-server/index.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { existsSync, lstatSync, readFileSync } from 'node:fs';
+import path from 'node:path';
+import process from 'node:process';
+import { LWCServer, LogLevel, ServerConfig, Workspace, startLwcDevServer } from '@lwc/lwc-dev-server';
+import { Logger } from '@salesforce/core';
+
+const DEV_SERVER_PORT = 8081;
```

Comment:
The port needs to be configurable. This a pretty popular port number and we should anticipate that it might now be available. Can we add a new flag to `sf lightning preview app` command?

Reply:
@ravijayaramappa we may need to run it by Marcelino to discuss this. For example:

- Should it be required or optional flag
- If it is optional and omitted what should we default too
- What if the user uses a `--port` flag to pass in a port but `lwc.config.json` also has a port defined in it. Should we update the content of the config file and overwrite with the new flag?
- etc.

Maybe we should utilize our slack channel to discuss further with him and the devs?

Reply:
I like that idea of just putting it in the `lwc.config.json`. So that the user can add it once and forget it. Created W-15962937 to track.